Kernel_23/include/CGAL/Origin.h

```diff
@@ -23,7 +23,18 @@
 namespace CGAL {
 
 class Origin
-{};
+{
+public:
+  const Origin& approx() const
+  {
+    return *this;
+  }
+
+  const Origin& exact() const
+  {
+    return *this;
+  }
```

Comment:
Lazy.h already contains
```c++
// For tag classes: Return_base_tag, Homogeneous_tag, Null_vector, Origin
template<class T>inline std::enable_if_t<std::is_empty<T>::value, T> exact(T){return {};}
```
Would it make sense, instead of adding this function, to change some `a1.approx()` to `CGAL::approx(a1)` where you were getting errors?